WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerViewModel.kt

```diff
@@ -111,13 +105,12 @@ class HomePagePickerViewModel @Inject constructor(
     }
 
     override fun onPreviewChooseTapped() {
-        super.onPreviewChooseTapped()
         onChooseTapped()
     }
 
     fun onChooseTapped() {
-        // TODO: adapt this to the new flow
         selectedLayout?.let { layout ->
+            super.onPreviewChooseTapped()
```

Comment:
@mkevins Delaying the super call that dismisses the preview screen and [unselects the layout](https://github.com/wordpress-mobile/WordPress-Android/blob/232a7307aedd27675262d5e29248a3d6d0f45ff4/WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt#L123) seems to fix the flow.
I removed the TODO but I'll be happy to put it back if there is more to it 🙏

Reply:
Fwiw, I added this comment before removing the bottom action buttons, and I left it in case anything else would need to be wired up when adding the new behavior. Delaying it sounds good to me (an alternative might be to pass it as a parameter, but I think this is simpler :man_shrugging: ).

One other consideration I had at the time was with regard to the tracks events, and whether we are differentiating between the user choosing from the main screen and the preview screen (in the old flow). In the new flow there is only one path to choose a theme, so maybe we just need to update the tracks description to reflect this?

Reply:
Thank you for the feedback @mkevins 🙇 

> One other consideration I had at the time was with regard to the tracks events, and whether we are differentiating between the user choosing from the main screen and the preview screen (in the old flow). In the new flow there is only one path to choose a theme, so maybe we just need to update the tracks description to reflect this?

That's a good point. I think with the old flow we do not differentiate between choosing directly or choosing in the preview screen and emit the same event `enhanced_site_creation_site_design_selected`. Despite that I think it's a good idea to make clear in the description of what we track when we update it to add the new `recommended` property.

ps. I added a draft issue `[Site Design Revamp] Update Tracks event (non-blocking)` on the board (cc @twstokes)